common/app/conf/switches/PerformanceSwitches.scala

```diff
@@ -148,8 +148,8 @@ trait PerformanceSwitches {
 
   val DiscussionSwitch = Switch(
     SwitchGroup.Performance,
-    "discussion",
-    "If this switch is on, comments are displayed on articles. Turn this off if the Discussion API is blowing up.",
+    "discussion-visible-web",
```

Comment:
I think its a good idea to have a UI switch, but it would probably cover at least most, if not all cases of comments showing. This would include comment count in fronts and the top of articles which currently is not the case.

Otherwise I would rename this to simply `comments-on-article`